scripts/start_gaudi_vllm_server.sh

```diff
@@ -149,9 +150,14 @@ case "$dtype" in
         echo Running with dtype="$dtype" ;;
     "fp8")
         echo Running with dtype="$dtype"
+	# TODO: quant config is different for different card number
         export QUANT_CONFIG=quantization/${model_name}/maxabs_quant_g2.json
         export PT_HPU_WEIGHT_SHARING=0
         QUANT_FLAGS=(--quantization inc --kv-cache-dtype fp8_inc)
+	model_basename=$(basename "${model_name}")
```

Comment:
`model_name` **is** `basename` as https://github.com/HabanaAI/vllm-fork/blob/7017e1da6b50dc4e3e9a7a0a0777cdf086bd559c/scripts/start_gaudi_vllm_server.sh#L110
You can simplify the extraction of `model_basename` there.